packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -0,0 +1,120 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.10;
+
+import { Ownable } from "../../lib/openzeppelin-contracts/contracts/access/Ownable.sol";
```

Comment:
To make this be able to compile with both hardhat and foundry when the deps are installed with foundry vs npm, not sure how to modify the remappings for hardhat but with foundry its in the `foundry.toml`

```
[default]
src = 'src'
out = 'out'
libs = ['lib']
remappings = [
    'ds-test/=lib/ds-test/src/',
]
```

I think this is fine and we don't need to mess with it unless we decide to completely ditch hardhat for this repo

Reply:
Hardhat is OK with the paths used above. 
However I did use this as a prompt to setup a basic `foundry.toml` file.